section1/set8/exercises/07-multiplication-table-exercises/02-math-tables/main.go

```diff
@@ -105,4 +113,57 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	if len(os.Args) != 3 {
+		fmt.Println("Usage: [op = %*/+-] [size]")
+		return
+	}
+
+	os1 := strings.IndexAny(os.Args[1], "%*-/+")
+
+	if os1 == -1 {
+		fmt.Println("Invalid operator.\nValid ops are: %+-*/")
+		return
+	}
+
+	n, err := strconv.Atoi(os.Args[2])
+
+	if err != nil {
+		fmt.Println("Incorrect size value.\nUsage: [op = %*/+-] [size]")
+		return
+	}
+
+	fmt.Printf("%5s", os.Args[1])
+	for i := 0; i <= n; i++ {
+		fmt.Printf("%5d", i)
+	}
+	fmt.Println()
+
+	for i := 0.; i <= float64(n); i++ {
```

Comment:
looping should always be in integer;

Reply:
I need to use the variable i inside the loop; it also needs to be a float value

Reply:
i am using the floating point number because I need the variable precision for dividing numbers in a division table